10up/gutenbridge
# Feature/setup cypress for e2e testing

### Description of the Change

This PR adds Cypress for e2e testing, configured and with 2 tests set-up:

1. block.test - the test creates a new classic post with dummy content, opens the post and checks content has been converted to blocks
2. admin.test - the test logs in and deactivates and activates the plugin


Closes #65

### How to test the Change


1. Pull changes
2. Run `npm install`
3. Build assets: `npm run build`
4. Start local environment: `npm run env:start`
5. Open Cypress for testing: `npm run cypress:open`
6. Use Cypress GUI to run tests and verify they are working

### Changelog Entry

> Added -  e2e testing with Cypress


### Checklist:
